.pipelines/templates/.build-and-test-windows-vhd-template.yaml

```diff
@@ -57,6 +57,7 @@ stages:
 
   - stage: e2e_${{ parameters.stageName }}
     dependsOn: build_${{ parameters.stageName }}
+    condition: eq('${{ parameters.build }}', True)

```

Comment:
do you need the condition if you already have `dependsOn`?

Reply:
It turns out you are right and moving the condition for the build stage is sufficient. The prev placement of the build condition wasn't working. But I quite like having both.